pkg/nmi/server/server.go

```diff
@@ -8,33 +8,44 @@ import (
 	"net"
 	"net/http"
 	"os"
-	"os/signal"
+
 	"regexp"
 	"runtime"
 	"runtime/debug"
 	"strconv"
 	"strings"
-	"syscall"
+
 	"time"
 
+	aadpodid "github.com/Azure/aad-pod-identity/pkg/apis/aadpodidentity"
 	auth "github.com/Azure/aad-pod-identity/pkg/auth"
 	k8s "github.com/Azure/aad-pod-identity/pkg/k8s"
 	"github.com/Azure/aad-pod-identity/pkg/metrics"
 	"github.com/Azure/aad-pod-identity/pkg/nmi"
-	"github.com/Azure/aad-pod-identity/pkg/nmi/iptables"
 	"github.com/Azure/aad-pod-identity/pkg/pod"
 	"github.com/Azure/go-autorest/autorest/adal"
+	v1 "k8s.io/api/core/v1"
```

Comment:
>v1 "k8s.io/api/core/v1" [](start = 1, length = 23)

This package will have name "v1" no need to define same alias. Did you want to define `corev1`?

Reply:
Oh, I see this is in preexisting code. hm.. feel free to ignore this.

---
In reply to: [383469572](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r383469572) [](ancestors = 383469572)

Reply:
yes. I keep it consistent with the preexisting code.

---
In reply to: [383471881](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r383471881) [](ancestors = 383471881,383469572)